test/lint/newline-after-block.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh -eu
+
+echo "Checking that functional blocks are followed by newlines..."
+
+files=$(find ./ -type f -name "*.go")
```

Comment:
`git ls-files '*.go'` is much faster here (on NVMe). Differences worth mentioning: the output produced is ordered differently and the `git` version doesn't have the leading `./`.

Reply:
Hrm, this wouldn't work if manually running `make static-analysis` on uncommited files though right?